app/Observers/CommentObserver.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Observers;
+
+use App\Models\Comment;
+use Illuminate\Support\Facades\Auth;
+
+class CommentObserver
+{
+    /**
+     * Handle the "saving" event.
+     *
+     * @param  Comment $comment
+     * @return void
+     */
+    public function saving(Comment $comment) {
```

Comment:
Interesting use of the observer. Is there a reason you chose to do this instead of simply setting the `user_id` in the `store` method of  `CommentsController`? For maintainability I think we should avoid using the observer pattern here unless necessary.

Reply:
I don't remember the exact reason, but there was one :thinking:

Reply:
I see what you mean, done.